src/hpke/mod.rs

```diff
@@ -373,7 +424,7 @@ mod tests {
                 let mut suite = EncryptionSuite::new(10, rng.clone());
                 // keep the originals, in case if we need to damage them
                 let (mut mkp_clone, mut site_domain_clone, mut helper_clone) = (mkp_origin.clone(), site_domain.clone(), helper_origin.clone());
-                let info = Info::new(0, 0, &mkp_origin, &site_domain, &helper_origin).unwrap();
+                let info = Info::new(0, 0, EventType::Source, &mkp_origin, &site_domain, &helper_origin).unwrap();
```

Comment:
this validates that source events will fail to decrypt with trigger bits, but does not validate the opposite. 

We can do both and it is easy with proptest

* generate a random bit
```rust
fn arbitrary_info_corruption(corrupted_info_field in 1..6,
                                         mkp_origin in "[a-z]{10}",
                                         site_domain in "[a-z]{10}",
                                         helper_origin in "[a-z]{10}",
                                         trigger_bit in 0_u8..=1, // <---
                                         seed: [u8; 32]) {
```

* convert this bit to event type

```rust
let event_type = EventType::try_from(trigger_bit).unwrap();
```

* flip the bit to induce trigger bit corruption

```rust
3 => Info {
    event_type: EventType::try_from(trigger_bit ^ 1).unwrap(),
    ..encryption.info
},

```

Reply:
okay, I added this to the arbitrary_info_corruption test and I think fixed the format issue